### PR TITLE
Consistently use PowerMode instead of boolean

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -481,10 +481,7 @@ where
     ) -> Result<(), Error<E>> {
         // Request measurement
         self.send_command(Command::Measure {
-            low_power: match mode {
-                PowerMode::LowPower => true,
-                PowerMode::NormalMode => false,
-            },
+            power_mode: mode,
```
Comment (dbrgn) [inline src/lib.rs]: If you name the parameter `power_mode` as well, you don't need to use `: mode`.

**Comment by dbrgn** [comment]
You can ignore Codecov for now.

File: src/lib.rs (hunk 1 above)
Comment (rnestler) [reply src/lib.rs]: Fixed.

**Review by dbrgn — COMMENTED** [review]
:+1: You can squash-merge once the tests pass :)

**Pull request merged**